### PR TITLE
Feat cities list

### PR DESCRIPTION
**Pull request by mikevic**

### CODE REVIEW

File: app/graphql/civis_api_schema.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+GraphQL::Field.accepts_definitions is_public: GraphQL::Define.assign_metadata_key(:is_public)
```
Comment (anbublacky) [inline app/graphql/civis_api_schema.rb]: is_public method seems to be not added in graphql controller

File: app/graphql/queries/constant_query_type.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Queries::ConstantQueryType = GraphQL::InterfaceType.define do
+  name "ConstantQuery"
+
+	field :constantByType, 							types[Types::Objects::ConstantType] do
+		is_public true
+	  description "Search for constants by supplying a type"
+	  argument :type,							!types.String
```
Comment (anbublacky) [inline app/graphql/queries/constant_query_type.rb]: This can be int right?

Comment (anbublacky) [reply app/graphql/queries/constant_query_type.rb]: Also type_id

File: app/graphql/queries/constant_query_type.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	  resolve ->(_, args, _) {
+	  	Constant.where(type_id: args[:type])
+	  }
+	end
+
+	field :constantHashMap,							Types::Objects::JsonType do 
+		is_public true
+		description "Returns a hash map for all constants on the system"
+		resolve -> (_, _, _){
+			Constant.hash_map
```
Comment (anbublacky) [inline app/graphql/queries/constant_query_type.rb]: This method does not seem to be present in constant.

File: app/graphql/types/objects/constant_type.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Types::Objects::ConstantType = GraphQL::ObjectType.define do
+  name "ConstantType"
+
+  field :id,                types.Int
```
Comment (manikandan0603) [inline app/graphql/types/objects/constant_type.rb]: Alphabetical order here.

File: app/graphql/civis_api_schema.rb (hunk 1 above)
Comment (mikevic) [reply app/graphql/civis_api_schema.rb]: @anbublacky - it is present in the `before action` and processed in another method.

**Comment by mikevic** [comment]
@anbublacky  @manikandan0603 - closing this PR, all fixed are in #3

**Pull request closed without merging**